package.json

```diff
@@ -25,11 +25,11 @@
   "dependencies": {
     "lodash": "^2.4.1",
     "rest": "^1.2.0",
-    "skynet": "^1.3.0"
+    "meshblu": "*"
```

Comment:
Some like `"meshblu": "^1.30.1"`
I just don't wan't it to automatically pull version 2 if there's breaking api changes.